Q4.js

```diff
@@ -0,0 +1,37 @@
+// Create a promise that makes a fetch call, but resolves with the data only 2 seconds after
+// the data has been received in the fetch.
+
+// Complete the above tasks with async/await. 
+
+
+async function fetch(url) {
+    try {
+      const response = await fetch(url);
+      
```

Comment:
YOu can't use a method as a function. 
Fetch is js default function. This will throw error, pls check